src/components/Content/Content.styled.ts

```diff
@@ -34,72 +55,44 @@ export const StyledEditIcon = styled(EditIcon)`
 
 export const ButtonsWrapper = styled.div`
 	align-items: center;
-	display: flex;	
+	display: flex;
 
 	.MuiButton-root {
 		padding-left: 16px;
```

Comment:
The hover state looks weird when using padding because it makes the blue background to grow on the left side. I suggest changing it to margin-left: 8px (because the button already has 8 to have a total of 16)